core/ui.libs/src/utils/Queue.ts

```diff
@@ -61,6 +61,11 @@ export default class EvanQueue {
    * @param      {number}  version  current db version
    */
   async openDB(version?: number) {
+    if (queueDB) {
+      queueDB.close();
+      await new Promise((resolve) => setTimeout(resolve, 50));
```

Comment:
What's the 50 ms timeout for? :)

Reply:
When we directly want to open the db again, we will get the indexeddb blocked error. An a onclose event isn't fired, when we are closing the connection by ourself. As a workaround, added the timeout and reopen works ;)